build.gradle.kts

```diff
@@ -23,9 +23,9 @@ plugins {
     // Java support
     id("java")
     // Kotlin support
-    id("org.jetbrains.kotlin.jvm") version "1.8.10"
```

Comment:
Can you change this back to 1.8.10? The build seems to fail locally as well otherwise.

Reply:
It worked fine when I tested but I downgraded it to 1.8.10

Reply:
Build is green now, will merge and release